subgraphs/compound-forks/protocols/dforce/src/mapping.ts

```diff
@@ -3,7 +3,7 @@ import {
   BigInt,
```

Comment:
I know this is done by prettier but any chance we can revert them so keep this PR clean?

Reply:
fixed, didn't realize it touched files out of scope